src/main/java/app/gpx_animator/Configuration.java

```diff
@@ -487,9 +558,9 @@ public Builder photoDirectory(final String photoDirectory) {
         public Builder photoTime(final Long photoTime) {
             this.photoTime = photoTime;
             return this;
-        }
-
-        public Builder photoAnimationDuration(final Long photoAnimationDuration) {
+        }        
+              
+      public Builder photoAnimationDuration(final Long photoAnimationDuration) {
```

Comment:
Another formatting issue: Because of the modified whitespace it looks like you changed six lines of code.